src/code/z_fbdemo.c

```diff
@@ -45,74 +57,84 @@ void TransitionUnk_InitGraphics(TransitionUnk* this) {
     for (frame = 0; frame < 2; frame++) {
         this->frame = frame;
         vtx = (this->frame == 0) ? this->vtxFrame1 : this->vtxFrame2;
-        for (colTex = 0, col = 0; col < this->col + 1; colTex += 0x20, col++) {
-            for (rowTex = 0, row = 0; row < this->row + 1; row++) {
-                vtx2 = &vtx->v;
+        rowTex = 0;
+        for (row = 0; row < this->rows + 1; row++) {
+            colTex = 0;
+            for (col = 0; col < this->cols + 1; col++) {
+                Vtx_tn* vtxn = &vtx->n;
```

Comment:
( @Thar0 ) in `sTransTileSetupDL`, `gsSPClearGeometryMode(... | G_SHADE | ... | G_LIGHTING` uses "clear" so vertices are actually not lit, and also not shaded at all I guess (`G_SHADE` cleared too)
so idk if Vtx_t / Vtx_tn is more appropriate here

Reply:
oh yeah I can't read, I was not expecting it to be this weird I guess, in this case it doesn't matter ultimately.
I think I still prefer Vtx_tn though,
```
vtxn->n[0] = 0;
vtxn->n[1] = 0;
vtxn->n[2] = 120;
```
This makes slightly more sense as a normal I think (although notably it isn't normalized 🤷)